include/mbedtls/config.h

```diff
@@ -1007,18 +1031,27 @@
 /**
  * \def MBEDTLS_USE_PSA_CRYPTO
  *
- * Make the X.509 and TLS library use PSA for cryptographic operations, see
- * #MBEDTLS_PSA_CRYPTO_C.
+ * Make the X.509 and TLS library use PSA for cryptographic operations, and
+ * enable new APIs for using keys handled by PSA Crypto.
  *
- * Note: this option is still in progress, the full X.509 and TLS modules are
- * not covered yet, but parts that are not ported to PSA yet will still work
- * as usual, so enabling this option should not break backwards compatibility.
+ * \note Development of this option is currently in progress, and parts
+ * of the X.509 and TLS modules are not ported to PSA yet. However, these parts
+ * will still continue to work as usual, so enabling this option should not
+ * break backwards compatibility.
  *
- * \warning  Support for PSA is still an experimental feature.
- *           Any public API that depends on this option may change
- *           at any time until this warning is removed.
+ * \warning The PSA Crypto API is in beta stage. While you're welcome to
+ * experiment using it, incompatible API changes are still possible, and some
+ * parts may not have reached the same quality as the rest of Mbed TLS yet.
```

Comment:
Please move back to the previous wording of the warning about PSA. In Mbed Crypto, we don't advertise PSA Crypto itself as being experimental, only mbedtls_xxx-over-PSA. And this is Mbed Crypto, not Mbed TLS.

Reply:
This warning is about the  mbedtls_xxx-over-PSA option, so it should be advertised as being beta/experimental. I've reworded "as the rest of Mbed TLS yet" to "as Mbed TLS yet".

Reply:
This warning is not about “the PSA Crypto API”, it's about using that API underneath mbedtls_xxx APIs. Please revert the whole paragraph.

Reply:
I made a change to avoid documenting the PSA Crypto API as is in beta stage in documentation for MBEDTLS_PSA_CRYPTO_C (we still have this documented in our README and possibly other places)